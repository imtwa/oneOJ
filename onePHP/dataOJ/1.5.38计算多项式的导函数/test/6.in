9
13 1 8 5 1 14 5 13 13 