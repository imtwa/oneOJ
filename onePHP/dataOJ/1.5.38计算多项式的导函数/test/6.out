117 8 56 30 5 56 15 26 13 